### ISSUE TITLE
Support for "Computers"/"Devices"?

### ISSUE THREAD

**Issue by bnxdev** [henrybear327/Proton-API-Bridge#9]
Hi! This project and the rclone integration are amazing - currently testing it out as a backup option.

Do you have any plans to support the "Computers" section in Proton Drive? E.g. in the Windows Desktop client it's the "My Computer" section that allows you to set up one-way sync of any folder.

I have no idea how they're storing those files and folders on Proton Drive itself - but the reason I ask is that this option provides (for me anyway) the best way to use Proton as a backup device. In my case I'd like to back up multiple-TB of photos and videos, without syncing these to all my computers. By using "My Computer" I can perform a one-way sync and only browse/download these on other computers if necessary - it helps keep things organised.

I've already got some Computers setup this way but cannot see them in any of the various `rclone lsd` type commands - so I feel this might be quite a complicated task.

Apologies as I can't contribute this feature myself (I don't know Go) but if there's any other way to support this project and your work please let me know! (Unless I'm blind I can't see any sponsor links?)

Many thanks!

**Comment by henrybear327** [comment]
Hi @bnxdev, 

Nice to e-meet you!

Great idea! I am currently wrapping up my master's thesis this week, so I won't have time to look into it this week, but I will try to look into the API route that they are using for this "Computer" section, as Rclone is indeed good at doing one-way sync!

One quick question, would you be able to test out the feature when I have a somewhat working version next week :) 

Also, thanks for the good idea of adding a sponsor link that I have never thought of before :D

Henry

**Comment by bnxdev** [comment]
Yes - absolutely - I'll definitely help test! I have a lot to back up (it should be a good stress test). Take your time, I'm in no rush, good luck with the thesis, and I'll keep a look-out for any changes! Thanks

**Comment by Pietro395** [comment]
I would add that it would be very useful to support 'Photos' as well, currently there is no client except mobile ones that can sync photos massively and the web interface has problems

@henrybear327 it you need help testing this i'm available

**Comment by bingoxo** [comment]
can you retake a look at this please
